### ISSUE TITLE
`skill:getParam()` nil errors in console log

### ISSUE THREAD

**Issue by TeoTwawki** [DarkstarProject/darkstar#3903]
**_I have:_**

- [x] searched existing issues (http://github.com/darkstarproject/darkstar/issues/) to see if the issue I am posting has already been addressed or opened by another contributor
- [x] checked the commit log to see if my issue has been resolved since my server was last updated

**_Client Version_** (type `/ver` in game) **:**
30170427_0

**_Server Version_** (type `@revision` in game) **:**
7fdf3f54834311fbe3080654bb57b2f9908a007e

**_Source Branch_** (master/stable) **:**
master

**_Additional Information_** (Steps to reproduce/Expected behavior) **:**
Other than being absurdly bad copy paste jobs the scripts in which this happens are not wrong. I mean, the script may suck but there is no readily apparent reason for that to be nil unless the binding is simply broken.
```lua
local potency = skill:getParam();
```
should be returning whatever value the skill's sql param actually has (which incidentally is also going to be just copy pasta - we have some mobskills that are very obviously not retail correct as far as potency right now).

```cpp
inline int32 CLuaMobSkill::getParam(lua_State* L)
{
    DSP_DEBUG_BREAK_IF(m_PLuaMobSkill == nullptr);

    lua_pushinteger(L, m_PLuaMobSkill->getParam());
    return 1;
}
```

Looks legit to me. \\\_(o.O)\_/

**Comment by teschnei** [comment]
Can you paste the whole message?

**Comment by TeoTwawki** [comment]
I swear I saw it in the console say specifically that the variable was nil..but now I can't find it. I just searched all my saved logs. 

I did find something related, which says that when I tried to do math and then clamp that same variable its telling me I tried to compare a number with nil inside utils.clamp it. utils.clap will error like that if fed bad values I think..so I'm not sure that really was the same as the glitch I saw before.

..Should I just close this till I can get the error again for sure? Sorry really thought I had a record of what stumped me here.
